tests/mapped_struct.py

```diff
@@ -1570,6 +1558,91 @@ def testIndexedCollectionEqual(self):
         self.assertNotEquals(c, pack([1, 2.0, 3]))
         self.assertNotEquals(c, pack([2, 2.0]))
 
+class ProxiedFrozensetPackingTest(unittest.TestCase, CommonCollectionPackingTest):
+    PACKING_CLASS = mapped_struct.proxied_frozenset
+    COLLECTION_CLASS = frozenset
+
+    def testContains(self):
+        values = [float(x) for x in range(5)]
+        c = self.pack(values)
+        for val in values:
+            self.assertIn(val, c)
+        for val in values:
+            self.assertNotIn(-val - 1, c)
+
+    def testCmpSets(self):
```

Comment:
Add a test case comparing with an invalid object and catch the exception.

Reply:
Python 2 doesn't raise an exception when comparing against non-sets, it just returns an arbitrary, but consistent value.